### PR TITLE
Added scripts in package.json to fix installation

### LINKED ISSUE #409: Fix installation setup by improving package.json.
The scripts in the package.json file are segregated incorrectly and this leads to unresolved warnings while trying to run the project locally. 

image

The unnecassary warnings prevent the project, from running locally. An updation of the Readme.md and package.json is required.

Note:I am working on this.

### PR DESCRIPTION
**Pull request by aashna27**
Fix #409 
Seperate scripts are added for setup and project start.These allow the project to run successfully. 
os:windows

### CODE REVIEW

**Comment by aashna27** [comment]
Yeahh , earlier what was happening was, that after the warnings, it was automatically exiting and the grunt server wasn't running, and now since we are running separately so the grunt server makes it run despite the  warnings and its not exiting.

**Comment by tech4GT** [comment]
Hmm, @aashna27 I think we need better integration with windows at a lot of other places as well, like test scripts. Maybe we can keep it as a long term issue. What do you think?
cc @jywarren

File: README.md [hunk 1]
```diff
@@ -45,9 +45,18 @@ A diagram of this running 5 steps on a single sample image may help explain how
 
 ## Installation
 
-This library works in the browser, in Node, and on the commandline (CLI), which we think is great. You can start a local environement to test the UI with `npm start`
+This library works in the browser, in Node, and on the commandline (CLI), which we think is great. You can setup a local environement to test the UI with `npm setup` followed by `npm start`
 On Unix based environments the command should be run as root user using `sudo npm start` and on windows the command should be run as administrator.
 
+Our npm scripts do not support windows shells, please run the following snippet in powershell.
+```powershell
```
Comment (jywarren) [inline README.md]: Could we have a subheader like `### Windows notes` here? Thanks!!!

Comment (tech4GT) [reply README.md]: Nice idea!

Comment (jywarren) [reply README.md]: @aashna27 what do you think, could we perhaps add this header in here? Thank you!

**Comment by tech4GT** [comment]
@jywarren How does this look?

**Comment by jywarren** [comment]
Perfect! Merging now!!! 🎉

**Pull request merged**